pkg/util/util.go

```diff
@@ -206,39 +205,64 @@ var IsPVCExcludedByLabel = func(namespace, pvcName string) (bool, error) {
 	return ok && label == "true", nil
 }
 
+func checkRestoreDataVolumePossible(backup *velerov1.Backup, namespace, name string) (bool, error) {
+	// IsDVExcludedByLabel first checks if DV exists
+	// If not no use of checking restore of DV
+	excluded, err := IsDVExcludedByLabel(namespace, name)
```

Comment:
Is excluded not always false? Or am I missing something? If it is, then there is no real reason to call this function and check the error or the excluded if statement.

Reply:
If DV was already GC so this will always return error not found and it will return. if no GC then its possible to have excluded label on the DV and also have the DV resource included or excluded from the backup. If the DV is GC we dont really care if the DV was or wasnt in the list as of included/excluded, so we return after we see the DV doesnt exist, and instead we check if the PVC is included/excluded from the backup

Reply:
But in this PR `IsDVExcludedByLabel` literally always returns false, nil. unless I am missing something.

Reply:
https://github.com/kubevirt/kubevirt-velero-plugin/pull/139/commits/49a8b141f0940b33f8fae074ab43db635e8da16f#diff-0fa449830b5dc2b274b7c95a58b5071edc23f1e44af84a99c1fa9e600767829fL168-R185
thats what it returns, it runs get DV if doesnt exist or error it return false, err other then that it checks for the label on the dv

Reply:
Wait I was looking at the _test go file, that always returns false. Nevermind.

Reply:
yeah I was just about to ask if you look at the UT :)